week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -22,23 +22,56 @@ function greaterThan10() {
   }
 }
 
+console.log(greaterThan10());
+
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
+  let sortedLetters = letters.sort();
+  /*
+  let i = 0;
+
+  //strSL = strSL.replace("'", '"');
+  while (sortedLetters[i]) {
+    sortedLetters[i] = ` \'${sortedLetters[i]}\'`;
 
+    if (i === sortedLetters.length - 1) {
+      let str1 = sortedLetters[i].toString;
+      sortedLetters[i] = str1.replace("'", "");
+      sortedLetters[i] = ` \'${sortedLetters[i]}\' `;
+    }
+
+    i++;
+  }
+
+  let strSL = sortedLetters.toString();
+*/
+  /*
+  My Array is create the same result as test but 
+  test has Failed !
+  */
   return sortedLetters;
+
+  // return `[${strSL}]`;
 }
 
+//to see what is the result
+console.log(sortArray());
+console.log(["a", "c", "e", "f", "n", "z"]);
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
-
+  let sliced = numbers.slice(0, 5);
+  /*
+  My Array is create the same result as test but 
+  test has Failed !
```

Comment:
Hey @babakjahangiri , was this addressed? Sorry if you posted on Slack and I missed this! I realise that this is becaues in the test I mistakenly equated two arrays, which will never be equal in JS. Apologies! Your approach is correct.